### ISSUE TITLE
az bot directline create fails to set--enable-enhanced-auth 

### ISSUE THREAD

**Issue by cxbuilders** [Azure/azure-cli#16659]
I know this is not about the docs itself, but I don't know where to submit an issue for an az bot command.

When using the following command:
az bot directline create --name myBot --resource-group myBot  --enable-enhanced-auth true --trusted-origins https://www.test1.com https://www.test2.com

The result is that the site is created but the isSecureSiteEnabled is false and the trustedOrigins are null:
      "sites": [
        {
          "isBlockUserUploadEnabled": false,
          "isEnabled": true,
          "isSecureSiteEnabled": false,
          "isV1Enabled": true,
          "isV3Enabled": true,
          "key": "xxxxx",
          "key2": "xxxxx",
          "siteId": "xxx",
          "siteName": "Default",
          "trustedOrigins": null
        }

When running the exact same command for the second time, everything is set correctly;
      "sites": [
        {
          "isBlockUserUploadEnabled": false,
          "isEnabled": true,
          "isSecureSiteEnabled": true,
          "isV1Enabled": true,
          "isV3Enabled": true,
          "key": "xxxxx",
          "key2": "xxxxx",
          "siteId": "xxx",
          "siteName": "Default Site",
          "trustedOrigins": [
            "https://www.test1.com",
            "https://www.test2.com"
          ]
        }
      ]

Basically it needs the command to run twice to get it right.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 71bc2cda-94fc-b5dc-210c-35c04c79aa97
* Version Independent ID: 42c469d4-88bb-4c44-d02b-b6e4c8efa7ba
* Content: [az bot directline](https://docs.microsoft.com/en-us/cli/azure/bot/directline?view=azure-cli-latest#az_bot_directline_create)
* Content Source: [latest/docs-ref-autogen/bot/directline.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/bot/directline.yml)
* Service: **bot-service**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
bot

**Comment by munozemilio** [comment]
Thanks for bringing this up @cxbuilders Im taking over the botservice component. I will asses and try to accommodate as soon as possible in the corresponding release.

**Comment by cxbuilders** [comment]
@munozemilio Do you know if around yesterday something has changed in the Bot Channels regsitration resource setup. We are running now into the following issue: #6203

**Comment by munozemilio** [comment]
@cxbuilders Im not aware on any change. @swagatmishra2007 do you know what could be the reason>

**Comment by swagatmishra2007** [comment]
i would need a botid to debug

**Comment by cxbuilders** [comment]
@swagatmishra2007 as mentioned in the other issue this has resolved itself. The original issue is still there, takes two arm / azure cli runs to create a direct line channel where the securesiteenabled is actually coming through

**Comment by swagatmishra2007** [comment]
that might be existing behavior ..i will bring it up for investigation and a possible fix
